demos/src/Examples/Messaging/React/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react'
+import { useEditor, EditorContent } from '@tiptap/react'
+import Document from '@tiptap/extension-document'
+import Paragraph from '@tiptap/extension-paragraph'
+import Text from '@tiptap/extension-text'
+import Placeholder from '@tiptap/extension-placeholder'
+import Bold from '@tiptap/extension-bold'
+import Italic from '@tiptap/extension-italic'
+import Strike from '@tiptap/extension-strike'
+import BulletList from '@tiptap/extension-bullet-list'
+import OrderedList from '@tiptap/extension-ordered-list'
+import ListItem from '@tiptap/extension-list-item'
+import Blockquote from '@tiptap/extension-blockquote'
+import Code from '@tiptap/extension-code'
+import CodeBlock from '@tiptap/extension-code-block'
+import Image from '@tiptap/extension-image'
+import Dropcursor from '@tiptap/extension-dropcursor'
+import Mention from '@tiptap/extension-mention'
+
+import mentionSuggestion from './mention-suggestion'
+import { content } from '../content'
+import TextformattingMenu from './TextformattingMenu'
+import './styles.scss'
+
+const CustomDocument = Document.extend({
+  addKeyboardShortcuts() {
+    return {
+      'Mod-Enter': () => {
+        document.getElementById('send').click()
+      },
+    }
+  },
+})
+
+export default () => {
+  const [json, setJson] = useState(null)
+  const editor = useEditor({
+    extensions: [
+      CustomDocument,
+      Paragraph,
+      Text,
+      Placeholder,
+      Bold,
+      Italic,
+      Strike,
+      BulletList,
+      OrderedList,
+      ListItem,
+      Blockquote,
+      Code,
+      CodeBlock,
+      Image,
+      Dropcursor,
+      Mention.configure({
+        HTMLAttributes: {
+          class: 'mention',
+        },
+        suggestion: mentionSuggestion,
+      }),
+    ],
+    content,
```

Comment:
I would add the standard content here directly instead of putting it in a separate file.